scipy/ndimage/tests/test_morphology.py

```diff
@@ -2257,6 +2257,61 @@ def test_binary_fill_holes03(self, xp):
         out = ndimage.binary_fill_holes(data)
         assert_array_almost_equal(out, expected)
 
+    @skip_xp_backends("cupy",
+                      reasons=["these filters do not yet have axes support"])
```

Comment:
could you also include `cpu_only=True, exceptions=['cupy', 'jax.numpy']` in your `skip_xp_backends` calls? Unfortunately, `skip_xp_backends` doesn't stack with itself, so you have to include all desired skip-flags at the level closest to the function/method. This will probably trip up with PyTorch on GPU, like gh-21564.

Reply:
Just added it, thanks. 

The test case that failed on "GCC Oldest" is this one, not sure if it has been seen in other MRs?

```
 =================================== FAILURES ===================================
______________________ interpolate/tests/test_interpnd.py ______________________
[gw0] linux -- Python 3.10.14 /opt/hostedtoolcache/Python/3.10.14/x64/bin/python
worker 'gw0' crashed while running 'interpolate/tests/test_interpnd.py::TestLinearNDInterpolation::test_threading'
```